app.py

```diff
@@ -51,35 +78,84 @@ def _existing_producer(queue: Queue, build_watcher_namespace: str):
         _LOGGER.debug("Listing tags available for %r", item["metadata"]["name"])
         for tag_info in item.status.tags or []:
             output_reference = f"{item.status.dockerImageRepository}:{tag_info.tag}"
+            _METRIC_IMAGES_SUBMITTED.inc()
```

Comment:
This is misleading - the event was just queued to internal build-watcher's queue for analysis, but it has not been submitted for analysis yet.

Reply:
Make sense, I would try to update it when the image is submitted for analysis.